mllib/src/main/scala/org/apache/spark/ml/feature/StringIndexer.scala

```diff
@@ -65,12 +66,15 @@ class StringIndexer(override val uid: String) extends Estimator[StringIndexerMod
   def this() = this(Identifiable.randomUID("strIdx"))
 
   /** @group setParam */
+  def setSkipInvalid(value: Boolean): this.type = set(skipInvalid, value)
```

Comment:
I'm probably just missing something but looking at the other params it seems that they are able to be set on the trainer and the mode so I figured I should keep with that style.